### ISSUE TITLE
WCAG 1.3.1: Ensure bold, italic text and font-size is not used to style <p> elements as a heading (.text-xl.mt-3.font-\[var\(--font-display\)\])

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#76]
#### Issue
`Ensure bold, italic text and font-size is not used to style <p> elements as a heading` ([p-as-heading](https://dequeuniversity.com/rules/axe/4.10/p-as-heading?application=msftAI))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/)
#### Element path
.text-xl.mt-3.font-\[var\(--font-display\)\]
#### Snippet
`<p class="mt-3 text-xl font-[var(--font-display)]">12/16</p>`
#### How to fix
    Fix all of the following:
      Unable to determine if <p> elements are styled as headings
#### Environment
Chrome version 143.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
